bindings/node.js/Dockerfile

```diff
@@ -1,6 +1,9 @@
 # Exists as a test harness for building and running tests in Linux
 
-FROM node:16
+
+FROM node:16-alpine
+RUN apk update && apk add --no-cache g++ make python3
```

Comment:
Makes the `make linux-test` command replicate EthereumJS's environment https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/Dockerfile#L1-L3

Reply:
This also makes explicit the build dependencies: a C/C++ compiler, make, and python3.